core/server/src/main/java/alluxio/master/file/FileSystemMasterClientRestServiceHandler.java

```diff
@@ -294,6 +301,67 @@ public Response mount(@QueryParam("path") String path, @QueryParam("ufsPath") St
     }
   }
 
+  private class MountPoint {
```

Comment:
a type sent over the wire should be defined under the alluxio.wire namespace with other wire types